static/styles/right_sidebar.css

```diff
@@ -319,3 +319,12 @@ $user_status_emoji_width: 24px;
         width: 90%;
     }
 }
+
+.status {
+    display: flex;
+
+    .status_emoji {
+        margin-right: 3px;
+        margin-left: 0;
+    }
+}
```

Comment:
Let's move this CSS under `#user_presences`, other than that looks good to me.

Reply:
@amanagr I have moved the CSS.

Reply:
Thanks!